src/sass/components/_header.scss

```diff
@@ -181,19 +182,56 @@ header {
         }
 
         i:last-child {
-          position: absolute;
-          right: 10px;
-          z-index: 0;
+          padding: 10px;
         }
 
-        select {
-          -webkit-appearance: none;
-          border: 0;
-          background-color: transparent;
-          outline: none;
-          padding: 5px 30px 5px 35px;
-          font-size: 14px;
-          position: relative;
+        ul {
+          margin: 0;
+
+          list-style-type: none;
+
+          a {
+            text-decoration: none;
+            font-size: 14px;
+            line-height: 45px;
+            color: $text-color;
+            white-space: nowrap;
+          }
+
+          ul {
+            display: none;
+
+            li {
+              display: block;
+              border-top: 1px solid $form-border-color;
+              height: 45px;
+
+              a {
+                display: block;
+                padding-left: 50px;
+                &:hover {
+                  color: white;
+                  background: $primary;
+                }
```

Comment:
Please make an empty line between the rules.

Reply:
ok. no problem!